src/main/java/uk/ac/ebi/eva/commons/models/converters/data/DBObjectToVariantSourceEntryConverter.java

```diff
@@ -0,0 +1,137 @@
+
+    private VariantStorageManager.IncludeSrc includeSrc;
+
+    private DBObjectToSamplesConverter samplesConverter;
+
+    /**
+     * Create a converter between VariantSourceEntry and DBObject entities when
+     * there is no need to provide a list of samples or statistics.
+     *
+     * @param includeSrc If true, will include and gzip the "src" attribute in the DBObject
+     */
+    public DBObjectToVariantSourceEntryConverter(VariantStorageManager.IncludeSrc includeSrc) {
+        this.includeSrc = includeSrc;
+        this.samplesConverter = null;
+    }
+
+
+    /**
+     * Create a converter from VariantSourceEntry to DBObject entities. A
+     * samples converter and a statistics converter may be provided in case those
+     * should be processed during the conversion.
+     *
+     * @param includeSrc       If true, will include and gzip the "src" attribute in the DBObject
+     * @param samplesConverter The object used to convert the samples. If null, won't convert
+     */
+    public DBObjectToVariantSourceEntryConverter(VariantStorageManager.IncludeSrc includeSrc,
+                                                 DBObjectToSamplesConverter samplesConverter) {
+        this(includeSrc);
+        this.samplesConverter = samplesConverter;
+    }
+
+    @Override
+    public VariantSourceEntry convertToDataModelType(DBObject object) {
+        throw new UnsupportedOperationException(
+                "This version of " + getClass().getSimpleName() + " is only for writing, try the version in OpenCGA");
+    }
+
+    @Override
+    public DBObject convertToStorageType(VariantSourceEntry object) {
+        BasicDBObject mongoFile = new BasicDBObject(FILEID_FIELD, object.getFileId())
+                .append(STUDYID_FIELD, object.getStudyId());
+
+        // Alternate alleles
+        if (object
+                .getSecondaryAlternates().length > 0) {   // assuming secondaryAlternates doesn't contain the primary alternate
+            mongoFile.append(ALTERNATES_FIELD, object.getSecondaryAlternates());
+        }
+
+        // Attributes
+        if (object.getAttributes().size() > 0) {
+            BasicDBObject attrs = null;
+            for (Map.Entry<String, String> entry : object.getAttributes().entrySet()) {
+                Object value = entry.getValue();
+                if (entry.getKey().equals("src")) {
+                    if (VariantStorageManager.IncludeSrc.FULL.equals(includeSrc)) {
+                        try {
+                            value = org.opencb.commons.utils.StringUtils.gzip(entry.getValue());
+                        } catch (IOException ex) {
+                            Logger.getLogger(DBObjectToVariantSourceEntryConverter.class.getName())
+                                  .log(Level.SEVERE, null, ex);
+                        }
+                    } else if (VariantStorageManager.IncludeSrc.FIRST_8_COLUMNS.equals(includeSrc)) {
```

Comment:
I'm not completely convinced about removing support for this, but well, removed in next commit